apps/remix-ide/src/app/editor/sourceHighlighter.js

```diff
@@ -39,6 +39,7 @@ class SourceHighlighter {
     this.source = null
     if (lineColumnPos) {
       this.source = filePath
+      if (!this.source) this.source = this._deps.fileManager.currentFile()
```

Comment:
Why do you need to do that? i'd say if `filePath` is not defined the function can't anyway process and should return an error or just return.